pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1835,9 +1867,9 @@ func parseConfigItems(config *zconfig.EdgeDevConfig, ctx *getconfigContext) {
 						agentName, current)
 				}
 			} else {
-				log.Errorf("Unknown configItem %s value %s\n",
+				errStr = fmt.Sprintf("Unknown configItem %s value %s\n",
 					key, item.Value)
-				// XXX send back error? Need device error for that
```

Comment:
Why remove comment?

Reply:
I added the support to store all the errors in globalConfig .. Hence this comment is no longer needed

In another PR - I added support to send back all errors from globalConfig to zedcloud in ZInfo msg..
That completes the feature..

We now need to change each item to add appropriate errors.. I put out a proposal to restructure this code to make it easy to add config items - to remove all the repetitive code.. Will take it all up as part of that..